### PR TITLE
feat: add legal instructions

### PR DESCRIPTION
**Pull request by lwywoo**
- [ ] Ready for review
- [ ] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?
Add legal instruction text to old and new formatters

#### What are the relevant tickets?
https://snyksec.atlassian.net/browse/BST-804
#### Screenshots
new formatter
Screenshot 2019-08-09 at 18 39 12
old formatter
Screenshot 2019-08-09 at 18 39 22

### CODE REVIEW

**Comment by lili2311** [comment]
screenshot please :)

File: test/acceptance/display-test-results.test.ts [hunk 1]
```diff
@@ -28,6 +28,43 @@ test('`test ruby-app` remediation displayed',  async (t) => {
...
+  chdirWorkspaces();
+  const stubbedResponse = JSON.parse(
+    fs.readFileSync(__dirname + '/workspaces/pip-app-license-issue/test-pip-stub-with-legal-instructions.json', 'utf8'),
+  );
+  const snykTestStub = sinon.stub(snyk, 'test').returns(stubbedResponse);
+  try {
+    await snykTest('pip-app-license-issue');
+  } catch (error) {
+    const res = error.message;
+    console.log(res);
```
Comment (lili2311) [inline test/acceptance/display-test-results.test.ts]: left over?

**Comment by lwywoo** [comment]
> screenshot please :)

Adding, need to rebase in some changes ....

File: test/acceptance/display-test-results.test.ts (hunk 1 above)
Comment (lwywoo) [reply test/acceptance/display-test-results.test.ts]: yup removed

**Comment by lwywoo** [comment]
Screenshot 2019-08-15 at 14 31 30
Screenshot 2019-08-15 at 14 31 41

**Comment by lwywoo** [comment]
Screenshot 2019-08-23 at 11 49 41
Screenshot 2019-08-23 at 11 41 42

**Pull request merged**